stubs/resources/js/Pages/Dashboard/DashboardIndex.vue

```diff
@@ -9,8 +9,76 @@
             <span class="font-bold">{{ $page.props.auth.user.name }}</span> !
         </div>
     </div>
-</template>
 
+    <div class="my-6 grid grid-cols-1 gap-6 md:grid-cols-3">
+        <!-- User Count Card -->
+        <div
+            class="shadow-xs flex items-center rounded-lg border border-skin-neutral-4 bg-skin-neutral-2 p-4 hover:cursor-pointer hover:bg-skin-neutral-1"
+            @click="$inertia.visit(route('user.index'))"
+            v-if="can('Acl')"
+        >
+            <div
+                class="mr-4 rounded-full bg-green-100 px-3 py-2 text-green-500 dark:bg-green-500 dark:text-green-100"
+            >
+                <i class="ri-user-fill text-2xl"></i>
+            </div>
+            <div>
+                <p class="mb-2 text-sm font-medium">Users</p>
+                <p class="text-lg font-semibold">
+                    {{ props.count['users'] }}
+                </p>
+            </div>
+        </div>
```

Comment:
Please extract to a dedicated DashboardCard.vue Component. Save it in `stubs/resources/js/Pages/Dashboard/Components/DashboardCard.vue`. Use this same component for all the 3 cards...